### PR TITLE
Detailing CBERS fields

### PR DESCRIPTION
**Pull request by fredliporace**
Related to https://github.com/stac-utils/stac-fields/issues/6#issuecomment-840651848

### CODE REVIEW

File: fields.json [hunk 1]
```diff
@@ -622,10 +622,23 @@
...
+			"label": "Data type",
+			"explain": "Geolocation precision level",
+			"mapping": {
+				"L2": "Geolocation using only satellite telemetry",
+				"L3": "Control points used to geolocate image, no terrain correction",
+				"L4": "Control points used to geolocate image, orthorectified"
+			},
+			"summary": false,
+		"cbers:path": {
+			"label": "CBERS Reference Grid Path",
```
Comment (m-mohr) [inline fields.json:635]:
```suggestion
			"label": "Reference Grid Path",
```

File: fields.json [hunk 2]
```diff
@@ -622,10 +622,23 @@
...
+			"summary": "r"
+		},
+		"cbers:row": {
+			"label": "CBERS Reference Grid Row",
```
Comment (m-mohr) [inline fields.json:639]:
```suggestion
			"label": "Reference Grid Row",
```

**Review by m-mohr — COMMENTED** [review]
Thanks a lot! Left some comments.

File: fields.json [hunk 3]
```diff
@@ -622,10 +622,23 @@
...
-
+		"cbers:data_type": {
+			"label": "Data type",
+			"explain": "Geolocation precision level",
+			"mapping": {
+				"L2": "Geolocation using only satellite telemetry",
+				"L3": "Control points used to geolocate image, no terrain correction",
+				"L4": "Control points used to geolocate image, orthorectified"
+			},
+			"summary": false,
```
Comment (m-mohr) [inline fields.json:633]: Why not summarize this field? I would think "values" could be appropriate to indicate at the collection-level which kind of data types are included?!

File: fields.json [hunk 4]
```diff
@@ -622,10 +622,23 @@
...
+		"cbers:path": {
+			"label": "CBERS Reference Grid Path",
+			"summary": "r"
```
[the remark this answers is not in the document]
Comment (fredliporace) [reply fields.json:636]: My suggestion is to keep as a way to confirm that this was reviewed and we confirmed that the fields are numerical.

File: fields.json (hunk 3 above)
Comment (fredliporace) [reply fields.json]: Using "v" now.

File: fields.json (hunk 1 above)
Comment (fredliporace) [reply fields.json]: done

File: fields.json (hunk 2 above)
Comment (fredliporace) [reply fields.json]: done

**Pull request merged**